test-data/unit/check-enum.test

```diff
@@ -6,10 +6,37 @@ class Medal(Enum):
     gold = 1
     silver = 2
     bronze = 3
+reveal_type(Medal.bronze)  # E: Revealed type is '__main__.Medal'
 m = Medal.gold
-m = 1
-[out]
-main:7: error: Incompatible types in assignment (expression has type "int", variable has type "Medal")
+m = 1  # E: Incompatible types in assignment (expression has type "int", variable has type "Medal")
+
+[case testEnumFromEnumMetaBasics]
+from enum import EnumMeta
+class Medal(metaclass=EnumMeta):
```

Comment:
This code doesn't actually work -- you need to also inherit from `enum.Enum`, else the class definition fails at runtime with
```
TypeError: Medal().__init__() takes no arguments
```
(Note: I'm not asking that mypy enforce this -- but I still think the test should work as an example.)

Reply:
I'm not sure what to do about it. If Medal inherits from `Enum`, the test doesn't check for what I want it to check. We might as well drop it.

Reply:
I can add a `def __init__(self, *args): pass` constructor, if you deem it better. It works, though might be considered an implementation detail.

Reply:
I'd say add a comment explaining that it doesn't work at runtime and what you are testing for.